TAScheduler/views_methods.py

```diff
@@ -61,45 +64,288 @@ def createSection(self, section_info):
         pass
 
     def removeCourse(self, active_course):
-        pass
+        if type(active_course) is not CourseObj:
+            raise TypeError("Input passed is not a Course object")
+        elif not Course.objects.filter(course_id=active_course.database.course_id).exists():
+            raise RuntimeError("Course does not exist")
+        Course.delete(active_course.database)
```

Comment:
I asked chatGPT if model.delete works and it said that "In Django, there is no built-in yourmodels.delete method. However, to delete objects (records) from a Django model, you can use the delete() method on a queryset." 

Example:
YourModel.objects.filter(some_field=some_value).delete()

Please let me know if this is still works.

Reply:
My tests pass with my implementation so what I did works